### PR TITLE
refactor:  application actor (DEV-956)

### PR DESCRIPTION
**Pull request by subotic**
Resolves DEV-956, DEV-300

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: DEV-956

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: k6/health.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import http from 'k6/http';
```
Comment (subotic) [inline k6/health.js:1]: Codacy has a fix for the issue: [Replace `'k6/http'` with `"k6/http"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
import http from "k6/http";
```

File: k6/health.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+import { sleep } from 'k6';
+export const options = {
+  vus: 10,
+  duration: '30s',
+};
+export default function () {
+  http.get('http://0.0.0.0:3333/health');
```
Comment (subotic) [inline k6/health.js:8]: Codacy has a fix for the issue: [Replace `'http://0.0.0.0:3333/health'` with `"http://0.0.0.0:3333/health"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
  http.get("http://0.0.0.0:3333/health");
```

File: k6/health.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
```
Comment (subotic) [inline k6/health.js:2]: Codacy has a fix for the issue: [Replace `'k6'` with `"k6"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
import { sleep } from "k6";
```

File: k6/health.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+export const options = {
+  vus: 10,
+  duration: '30s',
```
Comment (subotic) [inline k6/health.js:5]: Codacy has a fix for the issue: [Replace `'30s'` with `"30s"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
  duration: "30s",
```

File: webapi/src/test/scala/org/knora/webapi/CoreSpec.scala [hunk 5]
```diff
@@ -5,242 +5,117 @@
...
+   */
+  private val bootstrap: ZLayer[
+    Any,
+    Any,
+    Environment
+  ] = ZLayer.empty ++ Runtime.removeDefaultLoggers ++ SLF4J.slf4j ++ effectLayers
+
+  // add scope to bootstrap
+  private val bootstrapWithScope = Scope.default >>>
+    bootstrap +!+ ZLayer.environment[Scope]
```
Comment (irinaschubert) [inline webapi/src/test/scala/org/knora/webapi/CoreSpec.scala:62]: What does the `>>>` and the `+!+` mean/do?

File: webapi/src/test/scala/org/knora/webapi/core/TestClientService.scala [hunk 6]
```diff
@@ -260,10 +260,10 @@ object TestClientService {
    * Acquires a configured httpClient, backed by a connection pool,
    * to be used in communicating with SIPI.
    */
-  private def acquire(config: AppConfig) = ZIO.attemptBlocking {
+  private val acquire = ZIO.attemptBlocking {
 
     // timeout from config
-    val sipiTimeoutMillis = config.sipi.timeoutInSeconds.toMillis.toInt
+    val sipiTimeoutMillis = 120 * 1000
```
Comment (irinaschubert) [inline webapi/src/test/scala/org/knora/webapi/core/TestClientService.scala:266]: Why is this not taken from the config anymore?

**Review by irinaschubert — APPROVED** [review]

**Pull request merged**